Source/UpgradeScripts/v2_10_1RemovePayload.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using SaveUpgradePipeline;
+using UnityEngine;
+
+namespace RP0.UpgradeScripts
+{
+
+    [UpgradeModule(LoadContext.Craft, craftNodeUrl = "PART")]
+    public class v2_10_1RemovePayload : UpgradeScript
+    {
+        public override string Name { get => "RP-1 ComSatPayload and NavSatPayload Upgrader"; }
+        public override string Description { get => "Removes ComSatPayload and NavSatPayload from older Tanks"; }
+        public override Version EarliestCompatibleVersion { get => new Version(2, 0, 0); }
+        protected static Version _targetVersion = new Version(2, 10, 1);
+        public override Version TargetVersion => _targetVersion;
+
+        private TestResult TestRecurse(ConfigNode node)
+        {
+            ConfigNode.Value v = null;
+            if (node.name == "RESOURCE" && (v = node.values.values.Find(s => s.name == "name")) != null)
+            {
+                if (v.value == "ComSatPayload" || v.value == "NavSatPayload")
```

Comment:
Weathersat payload is removed above but there's no check for it here?

Reply:
Thanks, I'll add that check.